pom.xml

```diff
@@ -1,90 +1,150 @@
+
 -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <groupId>org.xwiki.platform</groupId>
-    <artifactId>xwiki-platform-core</artifactId>
-    <version>4.2</version>
-  </parent>
+    <modelVersion>4.0.0</modelVersion>
+    
+    <repositories>
+        <repository>
+            <id>repo-1</id>
+            <name>Main Repo</name>
+            <url>http://maven.xwiki.org/releases</url>
+        </repository>
+    
+        <repository>
+            <id>repo-2</id>
+            <name>Servlet Repo</name>
+            <url>http://repo.maven.apache.org/maven2</url>
+        </repository>
+        
+        <repository>
+            <id>repo-3</id>
+            <name>Servlet Repo</name>
+            <url>http://nexus.xwiki.org/nexus/content/repositories/externals</url>
+        </repository>
+        
+        <repository>
+            <id>repo-4</id>
+            <name>Servlet Repo</name>
+            <url>http://www2.ph.ed.ac.uk/maven2</url>
+        </repository>
+    </repositories>
+    
+    <parent>
+        <groupId>org.xwiki.platform</groupId>
+        <artifactId>xwiki-platform-core</artifactId>
+        <version>9.5.1</version>
+    </parent>
+
+    <groupId>org.xwiki.contrib.authentication</groupId>
+    <artifactId>xwiki-authenticator-jdbc</artifactId>
+    <name>XWiki Authenticator JDBC</name>
+    <version>1.1-SNAPSHOT</version>
+    <packaging>jar</packaging>
+    <description>XWiki authenticator based on custom database through JDBC API</description>
 
-  <groupId>org.xwiki.contrib.authentication</groupId>
-  <artifactId>xwiki-authenticator-jdbc</artifactId>
-  <name>XWiki Authenticator JDBC</name>
-  <version>1.1-SNAPSHOT</version>
-  <packaging>jar</packaging>
-  <description>XWiki authenticator based on custom database through JDBC API</description>
+    <properties>
+        <xwiki.checkstyle.skip>true</xwiki.checkstyle.skip>
+    </properties>
 
-  <properties>
-    <!-- Too lazy to fix them right now. -->
-    <xwiki.checkstyle.skip>true</xwiki.checkstyle.skip>
-  </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.xwiki.platform</groupId>
+            <artifactId>xwiki-platform-oldcore</artifactId>
+            <version>9.5.1</version>
+            <scope>provided</scope>
+        </dependency>
+                
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>servlet-api</artifactId>
+            <version>2.5</version>
+            <scope>provided</scope>
+        </dependency>
+        
+        <dependency>
+            <groupId>org.mindrot</groupId>
+            <artifactId>jbcrypt</artifactId>
+            <version>0.4</version>
+        </dependency>
+                
+    </dependencies>
+    
+    <build>
```

Comment:
As I said you should not embbed dependencies, the Extension Manager will deal with dependencies.

The pom.xml was fine before, you should probably just revert all the changes you made to this file and simply add your <dependency> since everything else does not have much to do with adding support for bcrypt.